### PR TITLE
Renamed Package

### PR DESCRIPTION
**Pull request by Matthew7758**
Tiger Tiger, burning bright,
In the forests of the night;
What immortal hand or eye,
Could frame thy fearful symmetry?

In what distant deeps or skies.
Burnt the fire of thine eyes?
On what wings dare he aspire?
What the hand, dare seize the fire?

And what shoulder, and what art,
Could twist the sinews of thy heart?
And when thy heart began to beat,
What dread hand? and what dread feet?

What the hammer? what the chain,
In what furnace was thy brain?
What the anvil? what dread grasp,
Dare its deadly terrors clasp!

When the stars threw down their spears
And water’d heaven with their tears:
Did he smile his work to see?
Did he who made the Lamb make thee?

Tiger Tiger burning bright,
In the forests of the night:
What immortal hand or eye,
Dare frame thy fearful symmetry?

### CODE REVIEW

File: .idea/dictionaries/entei.xml [hunk 1]
```diff
@@ -11,6 +11,8 @@
       <w>longname</w>
       <w>nodeid</w>
       <w>nodetype</w>
+      <w>numstaff</w>
```
Comment (ArchdukeTim) [inline .idea/dictionaries/entei.xml]: Stop committing things in the .idea folder

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This is the gradle configuration file. Modify this file to add dependencies to y
   modification you will need to make when you begin the project
   
   - `mainClassName`
-    - Modifiy this variable to point to your main class. By default it is `edu.wpi.teamname.Main
+    - Modifiy this variable to point to your main class. By default it is `Main
```
Comment (ArchdukeTim) [inline README.md]: the default is not Main

File: .idea/dictionaries/entei.xml (hunk 1 above)
Comment (Matthew7758) [reply .idea/dictionaries/entei.xml]: We don't know how to stop doing that.

File: README.md (hunk 2 above)
Comment (Matthew7758) [reply README.md]: Well that's what you gave us.

**Pull request merged**